tests/app/forms/validation/test_mutually_exclusive_validator.py

```diff
@@ -72,6 +72,7 @@ def test_mutually_exclusive_mandatory_answers_raise_validation_error(
         (["British, Irish", None], True),
         ([None, "I prefer not to say"], True),
         (["", "I prefer not to say"], True),
+        ([0, []], True),
```

Comment:
I would demo using `Decimal` class also.

Reply:
Added Decimal class to unit test.